zero/gkyl_gk_geometry.h

```diff
@@ -27,7 +27,9 @@ struct gk_geometry {
   // GK Equation and Poisson Equation and to apply certain BC's
   // The first 20 are defined on the configuration space domain. The last is a single element.
   struct gkyl_array* mc2p; // 3 components. Cartesian X,Y, and Z
+  struct gkyl_array* mc2p_reduced; // cdim components. Component removed (Z in 1x, R,Z in 2x, R,Z,phi in 3x)
   struct gkyl_array* mc2nu_pos; // 3 components. Uniform computational space to non-uniform computational space mapping
+  struct gkyl_array* mc2nu_pos_reduced; // cdim components. Uniform computational space to non-uniform computational space mapping
```

Comment:
I think these _reduced should be renamed _deflated. That's the terminology we typically use in the geo code for these sorts of things.